src/components/kubectl/port-forward.ts

```diff
@@ -210,6 +212,10 @@ export async function portForwardToPod (podName: string, portMapping: PortMappin
         } as portFinder.PortFinderOptions);
     }
 
-    kubectl.invokeInTerminal(`port-forward ${podName} ${usedPort}:${targetPort}`);
+    let usedNamespace = namespace === undefined ? 'default' : namespace;
+
+    kubectl.invokeInTerminal(`port-forward ${podName} ${usedPort}:${targetPort} -n ${usedNamespace}`);
```

Comment:
`kubectl.invokeInTerminal` will run the command in the shared terminal by default. But port forwarding plays as a proxy server, and the terminal will be always occupied. In order not to affect other operations, could we run `port-forward` in a new Terminal?